failed to bench antidote

from f8a7a83cc5a6e70c41a15d9f2230ce8d3eb5a37f

I have the following results when trying to bench antidote (same for riak). Am I missing somethin?

```bash
nherbaut@nherbaut-Precision-5520:~/workspace/FMKe$ make select-antidote
./scripts/config/change_db.sh antidote
Changed target_database to antidote.
success
nherbaut@nherbaut-Precision-5520:~/workspace/FMKe$ make bench-antidote
rm -rf _build/default/rel
rm -rf _build/default/rel/
rebar3 release -n fmke
===> Verifying dependencies...
===> Plugin protobuffs does not export init/1. It will not be used.
===> Plugin protobuffs does not export init/1. It will not be used.
===> Compiling fmke
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /home/nherbaut/workspace/FMKe/_build/default/lib
          /usr/local/lib/erlang/lib
===> Resolved fmke-0.1.0
===> Including Erts from /usr/local/lib/erlang
===> release successfully created!
./travis.sh bench normal antidote
> Setting up FMKe to benchmark antidote...
Changed target_database to antidote.
success
Changed database_ports to [8087].
make[1]: Entering directory '/home/nherbaut/workspace/FMKe'
rm -rf _build/default/rel
rm -rf _build/default/rel/
rebar3 release -n fmke
===> Verifying dependencies...
===> Plugin protobuffs does not export init/1. It will not be used.
===> Plugin protobuffs does not export init/1. It will not be used.
===> Compiling fmke
===> Starting relx build process ...
===> Resolving OTP Applications from directories:
          /home/nherbaut/workspace/FMKe/_build/default/lib
          /usr/local/lib/erlang/lib
===> Resolved fmke-0.1.0
===> Including Erts from /usr/local/lib/erlang
===> release successfully created!
make[1]: Leaving directory '/home/nherbaut/workspace/FMKe'
trying to load antidote docker image...
Using default tag: latest
latest: Pulling from mweber/antidotedb
Digest: sha256:e80e918e05525f79789ca78de48431eb7952e910c2d38b7e8d23036f5daeffd9
Status: Image is up to date for mweber/antidotedb:latest
antidote
antidote started.
compiling FMKe...
===> Verifying dependencies...
===> Plugin protobuffs does not export init/1. It will not be used.
===> Plugin protobuffs does not export init/1. It will not be used.
===> Compiling fmke
starting FMKe...
NODE_NAME: fmke@127.0.0.1
COOKIE: fmke
IP: 127.0.0.1
HTTP_PORT: 9090
waiting for FMKe to start...
FMKe started.
\033[32m> Populating antidote via FMKe using standard benchmark configuration...\033[0m\n
./scripts/populate_fmke.escript:177: syntax error before: 'case'
escript: There were compilation errors.
Makefile:17: recipe for target 'bench-antidote' failed
make: *** [bench-antidote] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -33,6 +33,7 @@ bench-riak-norm: rel
 
 compile:
 	${REBAR} as test compile
+	escript -s scripts/populate_fmke.escript
 
 console: rel
 	./_build/default/rel/fmke/bin/env console

--- a/scripts/populate_fmke.escript
+++ b/scripts/populate_fmke.escript
@@ -173,7 +173,7 @@ add_prescription_rec(Nodes, PrescriptionId, ListPatientIds, FmkConfig, Ops) ->
   PrescriberId = rand:uniform(FmkConfig#fmkeconfig.numstaff),
   Node = lists:nth(PrescriptionId rem length(Nodes) + 1, Nodes),
   OpArgs = [PrescriptionId, CurrentId, PrescriberId, PharmacyId, gen_random_date(), gen_random_drugs()],
-  Result = run_op(Node, create_prescription, OpArgs)
+  Result = run_op(Node, create_prescription, OpArgs),
   case divergence_failure(Result) of
       false ->
           ok;
@@ -207,10 +207,10 @@ run_op(FmkNode, create_prescription, Params) ->
 run_rpc_op(FmkNode, Op, Params) ->
   run_rpc_op(FmkNode, Op, Params, 0, ?MAX_RETRIES, none).
 
-run_rpc_op(_FmkNode, Op, Params, MaxTries, MaxTries, none) ->
+run_rpc_op(_FmkNode, _Op, _Params, MaxTries, MaxTries, none) ->
     io:format("FMKe node is reachable but operation timed out. Check status of FMKe node~n", []),
     {error, exceeded_num_retries};
-run_rpc_op(_FmkNode, Op, Params, MaxTries, MaxTries, Error) ->
+run_rpc_op(_FmkNode, _Op, _Params, MaxTries, MaxTries, Error) ->
     case Error of
         nodedown ->
             io:format("FMKe node is down or unreachable. Check link between populator and FMKe~n", []);
@@ -222,7 +222,7 @@ run_rpc_op(_FmkNode, Op, Params, MaxTries, MaxTries, Error) ->
             io:format("Connection between FMKe and DB is unstable/broken. Please check the link between FMKe and the database.~n", [])
     end,
     halt(2);
-run_rpc_op(FmkNode, Op, Params, CurrentTry, MaxTries, Error) ->
+run_rpc_op(FmkNode, Op, Params, CurrentTry, MaxTries, _Error) ->
     case rpc:call(FmkNode, fmke, Op, Params) of
         {badrpc, nodedown} ->
             run_rpc_op(FmkNode, Op, Params, CurrentTry + 1, MaxTries, nodedown);

